pythonarcade/arcade
# Fix GUI example docstrings

### Changes

Follow-up to #1750, prepares for further improvements to example docstring tests.

* Rename one of the GUI example files
* Add or correct top-level docstrings in GUI examples for clarity
* Ensure all docstrings include an example run command per project standards

### How to test

I know the instructions below are a pain. This PR is a step toward fixing the problem.

Locally retarget `test_example_docstrings.py` by making the following changes:
1. add `imnport arcade.gui` at the top
2. set `EXAMPLE_ROOT = "arcade.gui.examples"
3. In `test_docstrings`, replace `arcade.examples.__path__[0]` with `arcade.gui.examples.__path__[0]`
4. (Optional) if using PyCharm, make sure the run configuration  for this test sets the working directory to the project root
5. Run the test, either through the GUI or with `pytest tests/unit/test_example_docstrings.py`
